R/last_fit.R

```diff
@@ -71,6 +93,16 @@ last_fit.default <- function(object, ...) {
   rlang::abort(msg)
 }
 
+#' @export
+last_fit.model_fit <- function(object, ...) {
+  cli::cli_abort(c(
+    "{.help [{.fun last_fit}](tune::last_fit)} is not well-defined for \\
+     fitted model objects.",
+    "i" = "{.help [{.fun last_fit}](tune::last_fit)} takes \\
+           a model specification or unfitted workflow as its first argument."
```

Comment:
Should "model specification" and "unfitted workflow" also be linked to help-files?

Reply:
You got it. :) Added in [96ce9b3](https://github.com/tidymodels/tune/pull/675/commits/96ce9b3ebc8bd6d074f246a8fd9f2592fdc1a9e6).